Refine language for clarity in introduction section (#146)
myyoda/principles-paper: main.tex

\anchoredsection{introduction}{Introduction}

Data-driven research depends on specific versions of datasets, software, and computational environments, together with a record of how they were used to produce results.
We adopt the term \textbf{research object} to denote a collection of data, code, and metadata that together represent the research as a complete unit.

> It is common for the components of a research object to be organized independently with code in one repository, data on a shared drive, environment setup in a wiki page and provenance in a lab notebook.

Reviewer: ```suggestion
It is common for the components of a research object to be organized independently with code in one repository, data on a shared drive, environment setup in a wiki page, and provenance in a lab notebook.
```

"or nowhere at all" was maybe a little too negative and I almost removed it myself a couple times, but its unfortunately true. Anyway, I'm fine with removing, just add the oxford comma.

Author: Yeah I found it distracting when reading this on paper

This separation undermines rigor, reproducibility, reusability, and efficiency.

> Even when its components are gathered in one place, a research object is often tightly coupled to specific software versions opaque to outside collaborators, making it difficult to reuse or redistribute.

Reviewer: ```suggestion
Even when its components are gathered in one place, a research object is often tightly coupled to specific software versions and environments which may be opaque to outside collaborators, making it difficult to reuse or redistribute.
```

Optional: as you wrote is ok, but I think "environment" captures more, (like os), and "everything matters".

Even when its components are gathered in one place, a research object is often tightly coupled to specific software versions opaque to outside collaborators, making it difficult to reuse or redistribute.

The FAIR principles\cite{wilkinson_fair_2016} and FAIR4RS guidelines\cite{barker_introducing_2022} provide structured guidance for making digital objects "Findable, Accessible, Interoperable, and Reusable".
The Workflow Community Initiative (WCI-FW)\cite{wilkinson_applying_2025} mapped these principles onto computational workflows but explicitly chose not to define new ones, instead treating workflows as hybrid data \& software objects.
STAMPED addresses a complementary question: how a research object should be organized and managed so it can actually be re-executed, extended, and redistributed.
Both the FAIR and STAMPED frameworks cover overlapping concerns.
For example, persistent identifiers serve both Findability and Tracking but accent different aspects.
FAIR emphasizes discovery and interoperability across repositories, while STAMPED emphasizes improving the operational maturity of scientific workflows in day-to-day research practice.

> Many researchers already employ practices that address these operational properties, such as version control, containerized environments, structured project directories, and documented analysis steps.

Reviewer: ```suggestion
Many researchers already employ practices (such as version control, containerized environments, structured project directories, and documented analysis steps) that strengthen the operational properties of their research objects.
```

Idk I dont like my version either, but "these operational properties" bugs me, what are we referring to?

Author: properties relating to the operation of the research object, I imagine

Hopefully becomes clear later on or otherwise provides intrigue to keep reading

Not a fan of the parentheses for this specifically, trying to minimize the special grammars like that
